README.md

```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL. 
 - [owl2shacl](https://github.com/sparna-git/owl2shacl) - OWL-to-SHACL conversion rules.
+- [TopQuadrant: OWL Axioms to SHACL Constraints](https://www.topquadrant.com/doc/latest/reference/Transform_ConvertOWLAxiomsToSHACLConstraints.html) - Commercial closed-source solution by TopQuadrant
+- [elevont/owl2shacl](https://github.com/elevont/owl2shacl) - 
+[![License: AGPL-3.0-or-later](
```

Comment:
are we using images for licenses? All others use something like "`AGPL` license".